tests/testthat/test-main.R

```diff
@@ -0,0 +1,51 @@
+test_that('demandPriceTaker() works',{
+  population <- 31.99 # In millions
+  gdp_pc <- 6.71 # In thousand dollars
+  frac_high_risk <- 0.131
+  loss2yr <- 0.269
+  parameters <- Parameters$new(population=population, gdp_pc=gdp_pc, 
+                               frac_high_risk=frac_high_risk, loss2yr=loss2yr)
+  demand = demandPriceTaker(parameters)
+  expect_equal(class(demand),'list')
+  expect_length(demand,2)
+  opt = demand$optimizations
+  expect_true(all(opt>=0))
+  expect_is(demand$allCapacities,c("matrix", "array" ))
+  expect_true(all(demand$allCapacities>=0))
+  demand = demandPriceTaker(population = population, gdp_pc = gdp_pc, frac_high_risk = frac_high_risk, loss2yr = loss2yr)
```

Comment:
I think this block needs `skip_on_cran()` as we discussed, maybe also `skip_on_ci()`. CHeck examples online.